### PR TITLE
update Virt Launcher Pod Hypervisor Connection URI

### PR DESCRIPTION
**Pull request by akri3i**
Signed-off-by: akriti gupta <akrgupta@redhat.com>

##### More details:
This PR updates Virt Launcher Pod Hypervisor Connection URI socket to /var/run/libvirt/virtqemud.sock from /var/run/libvirt/libvirt-sock for 4.13

### CODE REVIEW

File: ocp_resources/virtual_machine_instance.py [hunk 1]
```diff
@@ -245,7 +245,7 @@ def virt_launcher_pod_hypervisor_connection_uri(self):
         return (
             ""
             if self.is_virt_launcher_pod_root
-            else "-c qemu+unix:///session?socket=/var/run/libvirt/libvirt-sock"
+            else "-c qemu+unix:///session?socket=/var/run/libvirt/virtqemud.sock"
```
Comment (myakove) [inline ocp_resources/virtual_machine_instance.py:248]: On which OpenShift version this was changed?

Comment (myakove) [reply ocp_resources/virtual_machine_instance.py:248]: Missed the commit msg :)

**Review by myakove — APPROVED** [review]

**Comment by akri3i** [comment]
!verified

**Comment by akri3i** [comment]
!do-not-merge

**Pull request closed without merging**